requirements.txt

```diff
@@ -17,7 +17,7 @@ numpy>=1.11.0,!=1.16.0
 Rtree>=0.8.2,!=0.9.1
 shapely>=2.0.0
 scipy>=1.9.0,!=1.12.*
-pygeoprocessing>=2.4.6  # pip-only
+pygeoprocessing>=2.4.6
```

Comment:
Building the pygeoprocessing wheel failed on linux, but we can install it from conda-forge.